tests/staking/createUnbondingtx.test.ts

```diff
@@ -27,7 +26,7 @@ describe.each(testingNetworks)("Create unbonding transaction", ({
     stakingTx,
     stakingOutputIndex: 0,
     startHeight: dataGenerator.getRandomIntegerBetween(
-      params.activationHeight, params.activationHeight + 1000,
+      700000, 800000,
```

Comment:
is it a good idea to stick with `700000, 800000`? Can we add a comment there or in the first example